### PR TITLE
Cleanup StringsHelper

### PR DESCRIPTION
**Pull request by cheenamalhotra**
- Remove unreferenced strings added in #1152 
- File touch-ups.

### CODE REVIEW

**Comment by Wraith2** [comment]
Nice. I like PR's which are deleting lots of things. 😁

**Comment by cheenamalhotra** [comment]
I would have liked it to be cleaned in the initial PR itself but I was too busy with other things, didn't get time to dig in review then.. so this one was on me!

File: src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs [hunk 1]
```diff
@@ -134,1041 +130,31 @@ internal partial class StringsHelper
...
-            internal const string DbParameter_IsNullable = @"a design-time property used for strongly typed code-generation.";
-            internal const string DbParameter_Offset = @"Offset in variable length data types.";
-            internal const string DbParameter_ParameterName = @"Name of the parameter.";
-            internal const string DbParameter_Size = @"Size of variable length data types (string & arrays).";
-            internal const string DbParameter_SourceColumn = @"When used by a DataAdapter.Update, the source column name that is used to find the DataSetColumn name in the ColumnMappings. This is to copy a value between the parameter and a data row.";
-            internal const string DbParameter_SourceVersion = @"When used by a DataAdapter.Update (UpdateCommand only), the version of the DataRow value that is used to update the data source.";
-            internal const string DbParameter_SourceColumnNullMapping = @"When used by DataAdapter.Update, the parameter value is changed from DBNull.Value into (Int32)1 or (Int32)0 if non-null.";
-            internal const string DbParameter_Value = @"Value of the parameter.";
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:198]: Could you include `DbParameter` const strings?

File: src/Microsoft.Data.SqlClient/netfx/src/Resources/StringsHelper.cs [hunk 2]
```diff
@@ -11,35 +11,22 @@ namespace Microsoft.Data
 {
     internal partial class StringsHelper : Strings
     {
-        static StringsHelper loader = null;
-        ResourceManager resources;
+        static StringsHelper s_loader = null;
+        readonly ResourceManager _resources;
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netfx/src/Resources/StringsHelper.cs:15]:
```suggestion
        private static StringsHelper s_loader = null;
        private readonly ResourceManager _resources;
```

File: src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs [hunk 3]
```diff
@@ -134,1041 +130,31 @@ internal partial class StringsHelper
...
-            internal const string DbTable_DeleteCommand = @"Used during Update for deleted rows in the DataTable.";
-            internal const string DbTable_InsertCommand = @"Used during Update for new rows in the DataTable.";
-            internal const string DbTable_SelectCommand = @"Used during Fill.";
-            internal const string DbTable_UpdateCommand = @"Used during Update for modified rows in the DataTable.";
-            internal const string DbTable_ReturnProviderSpecificTypes = @"Should Fill return provider specific values or common CLSCompliant values.";
-            internal const string DbTable_TableMapping = @"How to map source table to DataTable.";
-            internal const string DbTable_ConflictDetection = @"How are the Insert/Update/DeleteCommands generated when not set by the user.";
-            internal const string DbTable_UpdateBatchSize = @"Number of rows to batch together before executing against the data source.";
-            internal const string DbConnectionString_ConnectionString = @"The connection string used to connect to the Data Source.";
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:1159]: Could you include the `DbConnectionString_` const strings related to SqlClient?

File: src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs (hunk 1 above)
Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:198]: Did I miss - they are used anywhere?

File: src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs (hunk 3 above)
Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:1159]: These are unused strings.
SqlConnection_ConnectionString is used for instance.

File: src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs (hunk 1 above)
Comment (DavoudEshtehari) [reply src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:198]: My guess is they're for `SqlParameter`.

Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs:198]: I only found one "DbParameter_SourceColumnNullMapping" used in [SqlParameter](https://github.com/dotnet/SqlClient/blob/d1ff708949999bd8c02991beec19f3ac6c9540fe/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlParameter.cs#L840) which I'll include, but it's not used in NetFx and no other properties have description attributes, so there's inconsistency in there.

**Review by DavoudEshtehari — APPROVED** [review]

**Review by Kaur-Parminder — APPROVED** [review]

**Pull request merged**